pkg/tools/renderer/helpers.go

```
@@ -0,0 +1,454 @@
+				if strings.Contains(textLower, phrase) {
+					return true
+				}
+			}
+		}
+	}
+
+	return false
+}
+
+// calculateRiskLevel calculates overall risk level based on multiple factors
+func calculateRiskLevel(entry *Entry) string {
```

golangci-lint (1.64.8)

func `calculateRiskLevel` is unused

(unused)